src/main/java/io/kestra/plugin/aws/eventbridge/PutEvent.java

```diff
@@ -0,0 +1,147 @@
+package io.kestra.plugin.aws.eventbridge;
+
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.executions.metrics.Counter;
+import io.kestra.core.models.executions.metrics.Timer;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.aws.AbstractConnection;
+import io.kestra.plugin.aws.eventbridge.model.Entry;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import software.amazon.awssdk.http.apache.ApacheHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequest;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsResponse;
+
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Plugin(
+    examples = {
+        @Example(
+            title = "Put multiple event as map.",
+            code = {
+                "entries:",
+                "  - eventBusName: \"events\"",
+                "    source: \"Kestra\"",
+                "    detailType: \"my_object\"",
+                "    detail:",
+                "      message: \"hello from Event Bridge and Kestra\"",
+                "    resources:",
+                "      - \"arn:aws:iam::123456789012:user/johndoe\"",
```

Comment:
```suggestion
```

Given that this is an optional property that is rarely used, let's include it only in the second example 👍 the first one can show the simplest pattern